charmhelpers/core/hookenv.py

```diff
@@ -357,18 +358,16 @@ def _implicit_save(self):
 @cached
 def config(scope=None):
     """Juju charm configuration"""
```

Comment:
Opportunity to fill out this docstring.  What is scope (it's a str) and what does the function return depending on what scope is if it is set.